exporter/sumologicexporter/exporter.go

```diff
@@ -79,6 +92,71 @@ func newLogsExporter(
 }
 
 // pushLogsData groups data with common metadata and send them together to Sumo Logic
```

Comment:
Maybe update the doc to be more explicit what it returns?
Also mention there that it sends it in batches